plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMetadataFileOperations.java

```diff
@@ -384,10 +385,41 @@ public void testPredicateWithVarcharCastToDate()
         assertUpdate("DROP TABLE test_varchar_as_date_predicate");
     }
 
+    @Test
+    public void testRemoveOrphanFiles()
+            throws Exception
+    {
+        String tableName = "test_deleting_orphan_files_unnecessary_files" + randomTableSuffix();
```

Comment:
"test_remove_orphan_files_"?

or is the table name conveying more meaning that that?

Reply:
Nope, copy paste mess